<launch>
    <node name="vs_reactive_control_controller" pkg="vs_reactive_control_controller" type="vs_reactive_control_controller_node" output="screen">
        <rosparam file="$(find vs_reactive_control_controller)/param/param.yaml" />
        <remap from="lines" to="/line_events_tracker_lines" />
        <remap from="vel_cmd" to="/parrot/agiros_pilot/velocity_command" />
    </node>
</launch>